library/common/engine.h

```diff
@@ -74,8 +77,10 @@ class Engine {
 
 private:
   envoy_status_t run(std::string config, std::string log_level);
+  Stats::StatNameTagVector transform(const char* tag_arr[][2], uint64_t tag_arr_len);
```

Comment:
Move this to a static utility function similar to what we do with functions that transform `envoy_headers` to `HeaderMap`

Reply:
That also allows for unit testing this functionality.

Reply:
that sounds a good idea